src/redux/reducers/settings.js

```diff
@@ -33,12 +33,48 @@ const defaultState = {
   messageBackgroundImage: '',
   avatarType: 'default',
   devices: {},
+  userSettingsViewedByAdmin: {
```

Comment:
Instead of initialising with same values, create a `constant config` in the same file and use it for default state, userSettingsViewedByAdmin